stingray/utils.py

```diff
@@ -392,7 +396,7 @@ def rebin_data_log(x, y, f, y_err=None, dx=None):
     nsamples = np.array([len(binno[np.where(binno == i)[0]])
                          for i in range(1, np.max(binno) + 1, 1)])
 
-    return binx, biny, biny_err, nsamples
+    return binx_new, biny, biny_err, nsamples
```

Comment:
Minor comment: why was `binx` renamed here to `binx_new`? I guess this is a left-over from comparing the old `binx` to the new one? It'd be nice to keep the variable names consistent. :)

Reply:
`binx` is the list of input bins to all three rebinned quantities, this is a different quantity and it was better to call it differently. The alternative is preserving the `binx` name for the output, and rename the other quantity instead. No strong opinions

Reply:
I agree with Matteo, now in the routine there is binx which is the starting bin grid and binx_new the rebinned ones. Note that binx is different than the starting frequency grid because binx is the list of the edges of each frequency bin.

Reply:
My take is that I don't really mind what things are called inside, but it'd be nice to have a uniform user interface. It's kind of weird if the documentation says that the outputs are `binx_new` and `biny`. I'd vote to rather rename the variables inside to make the outputs consistent and easy to understand for users.

Reply:
Good point